examples/step-58/doc/intro.dox

```diff
@@ -0,0 +1,875 @@
+  \psi^{(3)}(t_n) &= \psi(t_n),
+@f}
+or equivalently,
+@f{align*}{
+  \psi^{(3)}(t_{n+1})
+  &=
+  \psi(t_n)
+  +
+  \int_{t_n}^{t_{n+1}}
+  \left(
+  -i\kappa |\psi^{(3)}(t)|^2 \,\psi^{(3)}(t)
+  \right)
+  \;
+  dt,
+@f}
+can be solved exactly: the equation is solved by
+@f{align*}{
+  \psi^{(3)}(t) = e^{-i\kappa|\psi(t_n)|^2 (t-t_{n})} \psi(t_n).
+@f}
+This is easy to see if (i) you plug this solution into the
+differential equation, and (ii) realize that the magnitude
+$|\psi^{(3)}|$ is constant, i.e., the term $|\psi(t_n)|^2$ in the
+exponent  is in fact equal to $|\psi^{(3)}(t)|^2$. In other words, the
+solution of the ODE for $\psi^{(3)}(t)$ only changes its <i>phase</i>,
+but the <i>magnitude</i> of the complex-valued function $\psi^{(3)}(t)$
+remains constant. This makes computing $I^{(3)}$ particularly convenient:
+we don't actually need to solve any ODE, we can write the solution
+down by hand. Using the operator splitting approach, none of the
+methods to compute $I^{(1)},I^{(2)}$ therefore have to deal with the nonlinear
+term and all of the associated unpleasantries: we can get away with
+solving only <i>linear</i> problems, as long as we allow ourselves the
+luxury of using an operator splitting approach.
+
+Secondly, one often uses operator splitting if the different physical
+effects described by the different terms have different time
+scales. Imagine, for example, a case where we really did have some
+sort of diffusion equation. Diffusion acts slowly, but if $\kappa$ is
+large, then the "phase rotation" by the term $-i\kappa
+|\psi^{(3)}(t)|^2 \,\psi^{(3)}(t)$ acts quickly. If we treated
+everything together, this would imply having to take rather small time
+steps. But with operator splitting, we can take large time steps
+$\Delta t=t_{n+1}-t_{n}$ for the diffusion, and (assuming we didn't
+have an analytic solution) use an ODE solver with many small time
+steps to integrate the "phase rotation" equation for $\psi^{(3)}$ from
+$t_n$ to $t_{n+1}$. In other words, operator splitting allows us to
+decouple slow and fast time scales and treat them differently, with
+methods adjusted to each case.
+
+
+<h3>Operator splitting: the "Lie splitting" approach</h3>
+
+While the method above allows to compute the three contributions
+$I^{(k)}$ in parallel, if we want, the method can be made slightly
+more accurate and easy to implement if we don't let the trajectories
+for the $\psi^{(k)}$ start all at $\psi(t_n)$, but instead let the
+trajectory for $\psi^{(2)}$ start at the <i>end point</i> of the
+trajectory for $\psi^{(1)}$, namely $\psi^{(1)}(t_{n+1})$; similarly,
+we will start the trajectory for $\psi^{(3)}$ start at the end point
+of the trajectory for $\psi^{(2)}$, namely $\psi^{(2)}(t_{n+1})$. This
+method is then called "Lie splitting" and has the same order of error
+as the method above, i.e., the splitting error is ${\cal O}(\Delta
+t)$.
+
+In formulas, this variation of operator splitting can be written as
```

Comment:
> In formulas, ...

Is the start of this sentence perhaps missing something?

Reply:
Or maybe its easier to write something like
"This variation of operator splitting can be formulated as follows..."

Reply:
I see that you've introduced some formulas several times in the same manner, so with that in mind I guess that you need not consider changing this if you don't feel like doing so.